packages/react-ui/src/components/HatchifyColumn/HatchifyColumn.tsx

```diff
@@ -39,26 +47,33 @@ export type RenderValue<
   >
 }) => JSX.Element
 
-// todo: renderValue and ValueComponent should be required, but only one can be provided
+// @TODO HATCH-459 - https://bitovi.atlassian.net/browse/HATCH-459 - renderDataValue and DataValueComponent should be required, but only one can be provided
+// @TODO HATCH-459 - https://bitovi.atlassian.net/browse/HATCH-459 - renderHeaderValue and HeaderValueComponent should be required, but only one can be provided
+// @TODO HATCH-459 - https://bitovi.atlassian.net/browse/HATCH-459 - renderHeaderValue, HeaderValueComponent, and label should all be optional, but only one can be provided
 export type AdditionalColumnProps<
   TSchemas extends Record<string, PartialSchema>,
   TSchemaName extends GetSchemaNames<TSchemas>,
 > = {
   allSchemas: FinalSchemas
   schemaName: string
   type: "append" | "prepend"
-  label: string
+  label?: string
```

Comment:
i think this label should still be required - what if someone adds an additional column but does not specify `renderHeaderValue` or `HeaderValueComponent` - we can still render a column with no label, but i think that it should be explicit from the user if they don't want to label their column

maybe we can avoid enforcing label if `renderHeaderValue` and `HeaderValueComponent` are omitted, but i think that logic will be caputured in hatch-459

Reply:
Fair points, reverted!